Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -486,6 +527,74 @@ def get_family_info_command(args: dict, intezer_api: IntezerApi) -> CommandResul
     )
 
 
+def submit_alert_command(args: dict, intezer_api: IntezerApi) -> CommandResults:
```

Comment:
Let's add optional source so we will be able to distinguish between xsoar and the original alert